apps/web/next-env.d.ts

```diff
@@ -3,4 +3,4 @@
 /// <reference types="next/navigation-types/compat/navigation" />
 
 // NOTE: This file should not be edited
-// see https://nextjs.org/docs/basic-features/typescript for more information.
+// see https://nextjs.org/docs/app/building-your-application/configuring/typescript for more information.
```

Comment:
Just making sure this was intentional, given the comment on line 5

Reply:
good catch, was not intented